### DISCUSSION TITLE
Save files on game server reboot

### DISCUSSION THREAD

**Discussion by mavproductions** [kus/cs2-modded-server#38] [Ideas]
Hey! Thanks for making cs2 mods easily accessible! Lovely mod.

I'm sure this is possible but I can't think off the top of my head how. When a server reboots, some files are wiped of their changes since the last reboot (nade lineups in prac mode).

I would like to snapshot those, and bring them over to the next server reboot. Any ideas how?

Thanks again,

**Comment by kus** [comment]
@mavproductions I just pushed an update where I deleted the default `savednades.json` so it won't overwrite your own saved nades anymore.

Please let me know if this now works for you, and also let me know if there are any other files that are being overwritten that it isn't easy to put in `custom_files` as they update a lot.
